<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tabs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1021</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="WorkArea">
     <attribute name="title">
      <string>Work Area</string>
     </attribute>
     <widget class="QTextEdit" name="Excerpts">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>591</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Rewrites">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>591</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="analysis">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>80</y>
        <width>331</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendopenai">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Send to AI</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="airesponse">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>971</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_excerpt">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Original Excerpt</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rewrite">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rewrite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_analysis">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>50</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Analysis</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_airesponse">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>450</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>AI Response</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_random">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Random</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Settings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QPlainTextEdit" name="apifield">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="apisave">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save API</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_prompt">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>103</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
